File: erettsegi_emelt/2013_may/c#/Szavazat.cs
public class Szavazat {
    public readonly int kerSzam;
    public readonly int szavazatok;
    public readonly string nev;
    public readonly string part;

    public Szavazat(string data) {
        var split = data.Split(' ');

        kerSzam = int.Parse(split[0]);
        szavazatok = int.Parse(split[1]);
        nev = split[2] + ' ' + split[3];
        part = split[4] == "-" ? "független" : split[4];
    }
}

File: erettsegi_emelt/2013_may/java/Szavazat.java
public class Szavazat {

    public final int kerSzam, szavazottSzam;
    public final String nev, part;

    public Szavazat(String data) {
        var split = data.split(" ");

        kerSzam = Integer.parseInt(split[0]);
        szavazottSzam = Integer.parseInt(split[1]);
        nev = split[2] + " " + split[3];
        part = split[4].equals("-") ? "Független" : split[4];
    }
}